spring-cloud-skipper-server-core/src/main/java/org/springframework/cloud/skipper/server/controller/ReleaseController.java

```diff
@@ -169,6 +170,18 @@ public ReleaseControllerLinksResource resourceLinks() {
 		return this.releaseResourceAssembler.toResource(release);
 	}
 
+	@RequestMapping(path = "/cancel/{name}", method = RequestMethod.POST)
+	@ResponseStatus(HttpStatus.OK)
+	public ResponseEntity<Void> cancel(@PathVariable("name") String releaseName) {
```

Comment:
Maybe we can use `CancelRequest` as a domain object and use it here for future expansion in this (similar to how we changed RollbackRequest).

Reply:
Yeah, that's probably a good idea.